src/python/grpcio/grpc/_cython/_cygrpc/channelz.pyx.pxi

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cdef class Channelz:
+
+    def GetTopChannels(self, start_channel_id):
+        cdef char *c_returned_str = grpc_channelz_get_top_channels(start_channel_id)
+        if c_returned_str == NULL:
+            raise ValueError('Failed to get top channels, please ensure your start_channel_id==%s is valid' % start_channel_id)
+        return c_returned_str
+    
+    def GetServers(self, start_server_id):
+        cdef char *c_returned_str = grpc_channelz_get_servers(start_server_id)
+        if c_returned_str == NULL:
+            raise ValueError('Failed to get servers, please ensure your start_server_id==%s is valid' % start_server_id)
```

Comment:
I'm still unclear on the `except` semantics of Cython. Will this propagate automatically as the function is declared `def` and not `cdef`? If not, would `except *` annotation be necessary?

Reply:
In my tests, under only one scenario will Cython ignore the exceptions in functions, it is `cdef` member function of a `cdef class` which is not returning Python object. So, it doesn't require `except *` here.